src/test/ui/consts/issue-54224.rs

```diff
@@ -1,6 +1,8 @@
+// compile-pass
+
 #![feature(nll)]
 
-const FOO: Option<&[[u8; 3]]> = Some(&[*b"foo"]); //~ ERROR temporary value dropped while borrowed
+const FOO: Option<&[[u8; 3]]> = Some(&[*b"foo"]);
```

Comment:
`*` used to not get promoted, because we didn't know references were guaranteed valid, but I think nowadays promotion is much stricter so that shouldn't be a problem anymore.

Reply:
Alright; we need to note this in the set of stabilization changes then if we permit this in stable. :)